adoc/Release-Notes.adoc

```diff
@@ -33,6 +33,174 @@ endif::[]
 [id='sec.major-change']
 == Major Changes
 
+[id='sec.1_5']
+=== Release 1.5, September 2019
+
+[id='sec.1_5.new']
+==== What Is New?
+
+* {scf} has been updated to version 2.18.0:
+** PXC (Percona XtraDB Cluster) replaces `cf-mysql` for database management -- please read the Known Issues section for this version on deployment and upgrade changes
+** Ability to set `config.HA_strict=false` in combination with `config.HA=true` to allow lowering the sizing count for a role below what is required for HA
+** UAA can now be embedded in the `cf` namespace -- please read the Known Issues section for this version on deployment limitations
```

Comment:
"UAA can now be deployed in one step when installing SCF, and embedded in the same namespace (not necessarily `cf`)"

Reply:
Reworded slightly to indicate the one step aspect.